src/main/java/net/snowflake/client/core/HttpUtil.java

```diff
@@ -617,28 +682,8 @@ public Socket createSocket(HttpContext ctx) throws IOException {
     }
   }
 
-  /** configure custom proxy properties from connectionPropertiesMap */
-  public static void configureCustomProxyProperties(
```

Comment:
No longer needed becauase static variables don't exist. Updated as part of the session instead.